products/ghc.md

```diff
@@ -0,0 +1,122 @@
+---
+title: GHC
+iconSlug: haskell
+category: lang
+tags: haskell-runtime
```

Comment:
We can add this later once we have enough haskell projects. 

```suggestion
```

From our [contribution guide](https://endoflife.date/contribute):


```
# Remember that no tag is better than a useless tag. So do not introduce new tags when adding a product
# and use one of the tags listed on https://endoflife.date/tags/.
#
# Should you want to add a new tag, please open an issue first to discuss it with the team.
# Moreover, any new tag must be applied in a single PR to all products that should have it.
```